src/widget/wsearchlineedit.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <qcompleter.h>
```

Comment:
why not `<QCompleter>` like the other Qt includes?

Reply:
This was what clangd auto-included it under, but I agree that `<QCompleter>` is cleaner and more consistent